src/main/java/seedu/contax/logic/parser/FindAppointmentCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package seedu.contax.logic.parser;
+
+import static seedu.contax.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.contax.logic.parser.CliSyntax.PREFIX_SEARCH_TYPE;
+
+import java.util.Arrays;
+
+import seedu.contax.logic.commands.FindAppointmentCommand;
+import seedu.contax.logic.parser.exceptions.ParseException;
+import seedu.contax.model.appointment.NameContainsKeywordsPredicate;
+import seedu.contax.model.appointment.PersonNameContainsKeywordsPredicate;
+
+
+
+
+/**
+ * Parses input arguments and creates a new FindAppointment object
```

Comment:
Small issue with javadocs.
```suggestion
 * Parses input arguments and creates a new FindAppointmentCommand object
```

Reply:
Fuxed